docker/docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       MYSQL_USER: '${DB_USERNAME:-dolibarr}'
       MYSQL_PASSWORD: '${DB_PASSWORD:-dolibarr}'
       MYSQL_ALLOW_EMPTY_PASSWORD: 'no'
+      TZ: '${TZ:-dolibarr}'
```

Comment:
Here we should provide the default value in case the env variable is not present.

```
TZ: '${TZ:-Europe/Paris}'
```

For example